testing/snapshot-test.md

```diff
@@ -0,0 +1,20 @@
+# Snapshot test
+
+## Introduction
+Snapshot tests are used to test visual regression of presentational components and for reducers in order to avoid deep comparison boilerplate code
+
+## Dos
+- The visual component output has to be relatively small.  
```

Comment:
Not very clear. I would phrase it something like - `Make sure that the snapshots are small and easy to read."